contributing-guides/style-guide.md

```diff
@@ -47,32 +47,43 @@ in order to allow `tldr` clients to highlight them.
 
 Keep the following guidelines in mind when choosing tokens:
 
-1. Use short but descriptive tokens,
-   ex. `{{source_file}}` or `{{wallet.txt}}`.
-2. Use [`snake_case`](https://en.wikipedia.org/wiki/Snake_case) for multi-word tokens.
-3. Use `{{filename}}` rather than `{{file_name}}`.
-4. For any reference to paths to files or directories, use the format `{{path/to/<placeholder>}}`.
-   For example, `ln -s {{path/to/file}} {{path/to/symlink}}`.
-   In case of a possible reference both to a file or a directory, use `{{path/to/file_or_directory}}`
-5. Follow the `{{path/to/<placeholder>}}` convention for all path-related commands, except when the
-   file location is implicit.
-6. If a command expects the file to have a particular extension, use it.
-   For example, `unrar x {{compressed.rar}}`.
-   In case a generic extension is needed, use `{{.ext}}`, but **only** if an extension is required.
-   For instance, in find.md's example "Find files by extension" (`find {{root_path}} -name '{{*.ext}}'`)
-   using `{{*.ext}}` explains the command without being unnecessarily specific;
-   But in a command like `wc -l {{file}}`, using `{{file}}` (without extension) is sufficient.
-7. If the example is clearer with an actual value rather than a generic placeholder, use the actual value.
-   For example, use `iostat {{2}}` rather than `iostat {{interval_in_secs}}`.
-8. If a command performs irreversible changes to a file system or to user's devices, then write every example in a way that they cannot be unmindfully copy-pasted by the user.
-   For example, instead of `ddrescue --force --no-scrape /dev/sda /dev/sdb` write `ddrescue --force --no-scrape {{/dev/sdX}} {{/dev/sdY}}` and use the `{{/dev/sdXY}}` placeholder for *block devices* instead of `/dev/sda1`.
+### Naming
+- Use short but descriptive tokens,
+  such as `{{source_file}}` or `{{wallet.txt}}`.
+- Use [`snake_case`](https://wikipedia.org/wiki/snake_case) for multi-word tokens.
+- Use an actual value rather than a generic placeholder where appropriate.
+  For example, use `iostat {{2}}` rather than `iostat {{interval_in_secs}}`.
+
+### Paths
+- Use `{{filename}}` rather than `{{file_name}}`.
+- For any reference to paths to files or directories, use the format `{{path/to/<placeholder>}}`.
+  For example, `ln -s {{path/to/file}} {{path/to/symlink}}`.
+  In case of a possible reference both to a file or a directory, use `{{path/to/file_or_directory}}`
+- Follow the `{{path/to/<placeholder>}}` convention for all path-related commands,
+  except when the file location is implicit.
```

Comment:
```suggestion
- For any reference to paths to files or directories, use the format `{{path/to/<placeholder>}}`, except when the file location is implicit.
  For example, `ln -s {{path/to/file}} {{path/to/symlink}}`.
  In case of a possible reference both to a file or a directory, use `{{path/to/file_or_directory}}`
```
I think we can merge those two